prov/gni/src/gnix_fabric.c

```diff
@@ -59,6 +59,19 @@
 const char gnix_fab_name[] = "gni";
 const char gnix_dom_name[] = "/sys/class/gni/kgni0";
 
+/* TODO: this will need to be adjustable - probably set in GNI_INI*/
+uint32_t gnix_def_max_nics_per_ptag = 4;
+/*
+ *  humph don't like this
+ */
+#if HAVE_ATOMICS
+atomic_t gnix_nics_per_ptag[GNI_PTAG_USER_END] = { {0} };
```

Comment:
If this is in global scope then you don't need the { {0} } as it will automatically be initialized to all 0.